index.html

```diff
@@ -1,110 +0,0 @@
-
-    <!-- Firebase Reference -->
-    <script src="https://www.gstatic.com/firebasejs/4.6.0/firebase.js"></script>
-
-    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
-  </head>
-
-  <body>
-
-      <br>
-
-      <!-- Jumbotron Title -->
-  <div class="container">
-
-      <div class="row">
-        <div class="panel grey">
-          <div class="jumbotron grey">
-              <h1 id="title">Welcome To Deal Finder</h1>
-              <h3 id="subtitle">Where Good Deals are Just a Click Away</h3>
-
-          </div>
-        </div>
-      </div>
-
-      <div class="row">
-        <div class="panel panel-default">
-          <div class="panel-body">
-
-            <div class="col-md-6" id="firstBox">
-
-              <form>
-
-                <div class="form-group">
-                  <label for="validationDefault01">Search Term</label>
-                  <input type="text" class="form-control user-search-term" id="validationDefault01" placeholder="Name of Service or Type of Business" required>
-                </div>
-
-                <br>
-
-                <div class="form-group" id="locationField">
-                  <label for="validationDefault02">Location</label>
-                  <input class="form-control user-location" id="autocomplete" placeholder="Street Address or City Name" type="text" required>
-                  <!-- onFocus="geolocate()" should be put to the left of the type="text" in the user-location input--> 
-                  <br>      
-                  
-                  <button class="btn grey" type="submit" id="search">Search</button>
-                </div>
-
-              </form>
-
-            </div>
-
-            <div class="col-md-6" id="secondBox">
-  
-
-            </div>
-
-
-            <div class="col-md-6 leftpad">
-
-              <div id="map_div">
-                
-              </div>
-          </div>
-  
-        </div>  
-      </div>
-    </div>
-    
-      <div class="row" id="deal-table-row">
-<!--       This is where the deals table is created using jquery.
- -->      
-      </div>
-
-     </div>
-
-
-
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
-
-    <script type="text/javascript" src="assets/js/sqoot.js"></script>
-    <script type="text/javascript" src="assets/js/googleMaps.js"></script>
-    <script type="text/javascript" src="assets/js/googleSearch.js"></script>
-    <script type="text/javascript" src="assets/js/firebase.js"></script>
-
-    
-    <script src="https://maps.googleapis.com/maps/api/js?v=3.exp&sensor=false&libraries=places"></script>
-    <script async src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBlo315C1BsvjDMw2FtAzOxvHz6YoBXXOA"></script>
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAvZV3rec51FwhSWqrrRkCySQw6h0cKfJk&libraries=places&callback=initAutocomplete"
```

Comment:
Google Maps API isn't quite happy that it was included 3 times. Find a way to work around this.